packages/x-components/src/components/result/base-result-image.vue

```diff
@@ -61,19 +71,51 @@
      */
     @Prop({ default: () => NoElement })
     public animation!: string | typeof Vue;
+
+    /**
+     * Indicates if the next valid image should be displayed on hover.
+     *
+     * @public
+     */
+    @Prop({ type: Boolean, default: () => false })
```

Comment:
We can simplify a bit this prop's definition:
```suggestion
    @Prop({ default: false })
```

Reply:
I would leave it as:

```
@Prop({ type: Boolean, default: false })
```

The `Boolean` type is needed there because of a bug in `vue-property-decorator`.

If you don't set it, when the component is used, and you set a boolean like `<component booleanProp />`, the value of `booleanProp` will be an empty string, instead of `true`.